src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/fonts.ts

```diff
@@ -0,0 +1,21 @@
+export const PrimaryFamily: string = 'Raleway, sans-serif';
```

Comment:
Same fir colors.ts, fonts.ts, typography.ts etc

Reply:
There were some changes in the headings and textlink. I created new files in the Molecules folder so that the existing code would not get affected.